### ISSUE TITLE
Not supported CDMA sim card

### ISSUE THREAD

**Issue by seasonyuu** [ohayoubaka/SamsungStatusBar#2]
![photo_2017-11-21_13-01-19](https://user-images.githubusercontent.com/8638535/33055712-ce4caa14-ce46-11e7-89bc-e8258991a31b.jpg)
![photo_2017-11-21_13-03-16](https://user-images.githubusercontent.com/8638535/33055740-037354ae-ce47-11e7-888b-28dfc76ee474.jpg)
As the title

**Comment by ohayoubaka** [comment]
I'll look into and try to add support for the top one, but the bottom one I can't really do I'm afraid.

**Comment by seasonyuu** [comment]
emmm... Can you just remove the bottom one?

**Comment by ohayoubaka** [comment]
I don't know if that's possible

**Comment by ohayoubaka** [comment]
Next update should fix this, wait for that and try again to see if your issue is resolved.

**Comment by ohayoubaka** [comment]
Try the new update and report back if your issue is fixed or not!

**Comment by seasonyuu** [comment]
![photo_2017-12-01_12-00-48](https://user-images.githubusercontent.com/8638535/33467425-569f4eb2-d68f-11e7-81a3-4558c89e740a.jpg)
It seems that nothing changed

**Comment by ohayoubaka** [comment]
Which carrier are you on, btw?

**Comment by seasonyuu** [comment]
China Telecom

**Comment by ohayoubaka** [comment]
Oh, so I did target the correct carrier icons. Hmm, running out of ideas...

**Comment by ohayoubaka** [comment]
Can you send me your SystemUI.apk?

**Comment by seasonyuu** [comment]
Here you are , hope that makes some helps
[SystemUI.apk from google drive](https://drive.google.com/file/d/1Vv2SueHzadIISGRRDuL95kkA1Vb_MELj/view?usp=sharing)

**Comment by ohayoubaka** [comment]
Thanks, I'll take a look into that systemui.apk and see what I can do.

**Comment by ohayoubaka** [comment]
Right, I've attempted to fix it again, wait for the next update coming very soon.

**Comment by ohayoubaka** [comment]
Try the latest update now, it should fix this issue for once and for all.

**Comment by seasonyuu** [comment]
Sorry, I hadn't check my GitHub notification these days.
When I update the latest version, I guess that it isn't work as same as you think.
![photo_2017-12-19_20-58-16](https://user-images.githubusercontent.com/8638535/34158196-6934743e-e4ff-11e7-86d7-94cdfeb77808.jpg)
![photo_2017-12-19_20-45-50](https://user-images.githubusercontent.com/8638535/34158212-749d099e-e4ff-11e7-85a1-80e2b6c86a69.jpg)

**Comment by ohayoubaka** [comment]
Well it's an improvement at least, signal icon themed. That 2G label is giving me trouble though...

**Comment by seasonyuu** [comment]
aha, your achievement looks very awesome, and thanks so much for your working

**Comment by mxpazyj** [comment]
I encountered the same problem
![image](https://user-images.githubusercontent.com/13087785/36237485-fb7811d4-1236-11e8-84f8-9c649dc5f3c9.png)
